data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/JacksonSpan.java

```diff
@@ -28,6 +28,8 @@
  */
 public class JacksonSpan extends JacksonEvent implements Span {
 
+    public static final String TRACE_GROUP_KEY = "traceGroup";
+    public static final String TRACE_GROUP_FIELDS_KEY = "traceGroupFields";
```

Comment:
Let's not expose these. Rather let's extend the `Span` interface to allow us to add these values. We can create two new methods `setTraceGroup` and `setTraceGroupFields`